src/components/VPopover/VPopover.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <div ref="triggerRef" class="w-min" @click="onTriggerClick">
+      <!--
+        @slot The trigger, should be a button 99.99% of the time. If you need custom event handling on the trigger button, ensure bubbling is not prevented or else the popover will not open
+          @binding {object} a11yProps
+          @binding {boolean} visible
+      -->
+      <slot
+        name="trigger"
+        :a11y-props="triggerA11yProps"
+        :visible="visibleRef"
+      />
+    </div>
+    <VPopoverContent
+      :visible="visibleRef"
+      :get-disclosure-element-ref="getDisclosureRef"
+      :final-focus-element="finalFocusElementRef"
+      :placement="placement"
+      :fixed="fixed"
+      :hide-visually="hideVisually"
+      :hide-on-esc="hideOnEsc"
+      :hide-on-click-outside="hideOnClickOutside"
+      :auto-focus-on-show="autoFocusOnShow"
+      :auto-focus-on-hide="autoFocusOnHide"
+      :gutter="gutter"
+      :hide="close"
+      :aria-label="label"
+      :aria-labelledby="labelledBy"
+    >
+      <!-- @slot The content of the popover -->
+      <slot name="content" />
```

Comment:
Since this doesn't appear to have a ref or any props passed to the slot, should we make this the _default_ slot to make the component api a little simpler? I think there's also something compelling about having the two explicit slots in the current implementation, so up to you.

Reply:
Yeah! We can do that.